proto/ibc/applications/fee/v1/fee.proto

```diff
@@ -32,7 +32,7 @@ message Fee {
 // the refund address to which any unused funds are refunded,
 // and an optional list of relayers that are permitted to receive the fee.
 message IdentifiedPacketFee {
-  ibc.core.channel.v1.PacketId packet_id      = 1 [(gogoproto.moretags) = "yaml:\"packet_id\""];
+  ibc.core.channel.v1.PacketId packet_id      = 1 [(gogoproto.nullable)     = false, (gogoproto.moretags) = "yaml:\"packet_id\""];
```

Comment:
The formatting on this line looks a little funky with extra whitespace. Maybe try run `make proto-format`?

Reply:
`make proto-format` made it look this way :) Agree it looks odd though.

Reply:
we can update the [proto format](https://github.com/cosmos/ibc-go/blob/main/.clang-format) settings. It was copied from the SDK